Warn about incompatibility of policy and materialized view (#1644)
MicrosoftDocs/dataexplorer-docs: data-explorer/kusto/management/restrictedviewaccesspolicy.md

> [!NOTE]
> RestrictedViewAccess policy can't be configured on a table on which a [Row Level Security policy](./rowlevelsecuritypolicy.md) is enabled.

> [!WARNING]
>

> > A table with RestrictedViewAccess policy enabled cannot be used as the source of a materialized view. See [materialized views limitations and known issues](./materialized-views/materialized-views-limitations#the-materialized-view-source) for more details.

Reviewer: ```suggestion
> A table with RestrictedViewAccess policy enabled cannot be used as the source of a materialized view. See [materialized views limitations and known issues](./materialized-views/materialized-views-limitations.md#the-materialized-view-source) for more details.
```

> > [!WARNING]
> >
> > A table with RestrictedViewAccess policy enabled cannot be used as the source of a materialized view. See [materialized views limitations and known issues](./materialized-views/materialized-views-limitations.md#the-materialized-view-source) for more details.

Reviewer 2: ```suggestion
> [!IMPORTANT]
>
> A table with RestrictedViewAccess policy enabled cannot be used as the source of a materialized view. For more information, see [materialized views limitations and known issues](materialized-views/materialized-views-limitations.md#the-materialized-view-source).
```

> > [!IMPORTANT]
> >
> > A table with RestrictedViewAccess policy enabled cannot be used as the source of a materialized view. For more information, see [materialized views limitations and known issues](materialized-views/materialized-views-limitations.md#the-materialized-view-source).

Reviewer: ```suggestion
> [!IMPORTANT]
> A table with RestrictedViewAccess policy enabled cannot be used as the source of a materialized view. For more information, see [materialized views limitations and known issues](materialized-views/materialized-views-limitations.md#the-materialized-view-source).
```
